.werft/jobs/build/prepare.ts

```diff
@@ -62,12 +61,12 @@ function configureStaticClustersAccess() {
     }
 }
 
-function issueCertificate(werft: Werft, config: JobConfig) {
+async function issueCertificate(werft: Werft, config: JobConfig) {
     const certName = config.withVM ? `harvester-${previewNameFromBranchName(config.repository.branch)}` : `staging-${previewNameFromBranchName(config.repository.branch)}`
     const domain = config.withVM ? `${config.previewEnvironment.destname}.preview.gitpod-dev.com` : `${config.previewEnvironment.destname}.staging.gitpod-dev.com`;
 
     werft.log(prepareSlices.ISSUE_CERTIFICATES, prepareSlices.ISSUE_CERTIFICATES)
-    issueMetaCerts(werft, certName, "certs", domain, config.withVM, prepareSlices.ISSUE_CERTIFICATES)
+    await issueMetaCerts(werft, certName, "certs", domain, config.withVM, prepareSlices.ISSUE_CERTIFICATES)
```

Comment:
Same as the comment above, this is just creating the Certificate, no need to use "fire-and-forget"

Reply:
👍